src/Stratis.Bitcoin.IntegrationTests/CoinViewTests.cs

```diff
@@ -172,9 +172,9 @@ public void CanHandleReorgs()
         {
             using (NodeBuilder builder = NodeBuilder.Create())
             {
-                var stratisNode = builder.CreateStratisPowNode();
-                var coreNode1 = builder.CreateNode();
-                var coreNode2 = builder.CreateNode();
+                var stratisNode = builder.CreateBitcoinPowNode();
+                var coreNode1 = builder.CreateBitcoinCoreNode();
+                var coreNode2 = builder.CreateBitcoinCoreNode();
```

Comment:
hmm so your solution is to default to a statis (c#) node and if we use core then just name it core hmmmm...

How about we actually also use core  for stratis and core will be the code name for the tech.
Then the token can follow


so it will be 
CreateStratisCoreBitcoinNode
CreateBitcoinCoreBitcoinNode

and for the stratis network 
CreateStratisCoreStratisNode
CreateBitcoinCoreStratisNode (this is invalid ofcourse unless we actually also use stratisx)

WDYT

@Aprogiena @bokobza thoughts?

Reply:
I would even go one step further and renaming the namespace to `Stratiscore.Bitcoin` instead of Stratis.Bitcoin`

Reply:
I am missing context, not sure what the question is ...

Reply:
@dangershony I reckon the node runners were initially called StratisBitcoinPowNode to follow the daemon's naming convention.

I only named it BitcoinCore because its the only runner which starts up a process too execute the Bitcoin.exe, the rest are all effectively Stratis nodes.

I think we should go with:

CreateBitcoinCore (Starts up the Bitcoin.exe in a process)
CreateStratisCorePowNode (Starts up a PoW Stratis node)
CreateStratisCorePosNode (Starts up a PoS Stratis node)
CreateStratisBitcoinPowNode (Starts up a Stratis node mining bitcoin)

Reply:
>I am missing context
We currently start two type of codebase the C++ bitcoincore and the C# stratis 
But we also have distinction of token, all of our tests currently use the bitcoin network, but we will be wanting to write tests for the stratis network as well.

But then again the distinction is more about POW/POS so fassa is sort of correct that we need to make the distinction like that.

But this is not so correct 
>CreateStratisCorePowNode (Starts up a PoW Stratis node)
>CreateStratisCorePosNode (Starts up a PoS Stratis node)

A POS node is always also a POW node how you choose to mine is up to you.

CreateBitcoincorePowNode (c++ on bitcoin network)
CreateStratiscorePowNode  (c# node bitcoin network)
CreateStratiscorePosNode   (c# node stratis network)